include/swift/Basic/Located.h

```diff
@@ -0,0 +1,39 @@
+//===--- Located.h - Source Location and Associated Value ----------*- C++ -*-===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2019 Apple Inc. and the Swift project authors
```

Comment:
Sorry for nitpicking but this year should be just 2019.

Reply:
Sure. Fixed.